daemon/src/rollover_taker.rs

```diff
@@ -200,30 +208,37 @@ impl Actor {
 
         match event_fut.await {
             Ok(event) => {
-                let _ = self
-                    .process_manager
-                    .send(process_manager::Event::new(event))
-                    .await;
+                if let Some(event) = event {
+                    let _ = self
+                        .process_manager
+                        .send(process_manager::Event::new(event))
+                        .await;
+                }
             }
             Err(e) => {
-                tracing::warn!(%order_id, "Failed to report completion of collab settlement: {:#}", e)
+                tracing::warn!(%order_id, "Failed to complete rollover: {:#}", e)
             }
         }
 
         ctx.stop();
     }
+
+    /// Returns whether the maker has accepted our rollover proposal.
+    fn is_accepted(&self) -> bool {
+        self.rollover_msg_sender.is_some()
```

Comment:
❓How is `accepted` ensured by the message *sender* being `Some`? I would have expected the message receiver being `Some` here, but that might just be a lack of understanding. Would be great to at least explain the reasoning in a comment.